helm-go-lib/main.go

```diff
@@ -91,19 +95,21 @@ func listHelm(namespace, kubeToken, apiServer string) {
 // TODO: Do we need to make 'overrideValues' optional?
 // TODO: If so, emulate it (perhaps via variadic functions) since Golang doesn't support optional parameters :(
 // `HELM_DRIVER` env variable is expected to have been set to the right value (which is likely to be "secret")
-func installChart(namespace string, kubeToken string, apiServer string, releaseName string, chartName string, overrideValues string) *C.char {
-	settings := cli.New()
+func installChart(namespace string, kubeToken string, apiServer string, caFile string, releaseName string, chartName string, overrideValues string) *C.char {
+	var kubeConfig *genericclioptions.ConfigFlags
```

Comment:
Building a `kubeConfig` manually instead of using `kube.GetConfig` which doesn't have an option for the ca cert and I _think_ just reads from `~/.kube/config`. 

Inspired from https://stackoverflow.com/questions/59782217/run-helm3-client-from-in-cluster and https://github.com/kubernetes/cli-runtime/blob/master/pkg/genericclioptions/config_flags.go#L80